### PR TITLE
skipper-internal: Update main fleet to version v0.21.224-1047

### PR DESCRIPTION
**Pull request by MustafaSaber**
* https://github.com/zalando/skipper/pull/3271
* https://github.com/zalando/skipper/pull/3283
* https://github.com/zalando/skipper/pull/3284
* https://github.com/zalando/skipper/pull/3285
* https://github.com/zalando/skipper/pull/3281
* https://github.com/zalando/skipper/pull/3282
* https://github.com/zalando/skipper/pull/3294
* https://github.com/zalando/skipper/pull/3293
* https://github.com/zalando/skipper/pull/3292
* https://github.com/zalando/skipper/pull/3291
* https://github.com/zalando/skipper/pull/3288
* https://github.com/zalando/skipper/pull/3300
* https://github.com/zalando/skipper/pull/3301
* https://github.com/zalando/skipper/pull/3290
* https://github.com/zalando/skipper/pull/3328
* https://github.com/zalando/skipper/pull/3319
* https://github.com/zalando/skipper/pull/3325
* https://github.com/zalando/skipper/pull/3310
* https://github.com/zalando/skipper/pull/3307
* https://github.com/zalando/skipper/pull/3322
* https://github.com/zalando/skipper/pull/3326
* https://github.com/zalando/skipper/pull/3289

https://github.com/zalando/skipper/compare/v0.21.220...v0.21.224

follow up on https://github.com/zalando-incubator/kubernetes-on-aws/pull/8574

### CODE REVIEW

**Comment by MustafaSaber** [comment]
👍

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -166,6 +166,8 @@ spec:
           - "-kubernetes-default-lb-algorithm={{ .Cluster.ConfigItems.skipper_ingress_default_lb_algorithm }}"
           - "-kubernetes-disable-catchall-routes={{ .Cluster.ConfigItems.skipper_ingress_disable_catchall_routes }}"
           - "-enable-kubernetes-endpointslices={{ .Cluster.ConfigItems.skipper_endpointslices_enabled }}"
+          - '-kubernetes-annotation-predicates={{ .Cluster.ConfigItems.skipper_kubernetes_annotation_predicates }}'
+          - '-kubernetes-east-west-range-annotation-predicates={{ .Cluster.ConfigItems.skipper_kubernetes_east_west_range_annotation_predicates }}'
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml]: Minor: maybe move these two after `-kubernetes-east-west-range-predicates` for better grouping.

**Comment by demonCoder95** [comment]
👍

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**